server/test/src/main/java/org/eclipse/lsp/cobol/test/engine/UseCaseEngine.java

```diff
@@ -278,6 +280,7 @@ public AnalysisResult runTest(
                 .build(),
             languageId);
     assertResultEquals(actual, document.getTestData());
+    UseCaseUtils.storeDocumentToUnitTextExtensionContext(document.getText(), document.getCopybooks(), document.getTestData());
```

Comment:
Isn't this fuction called by `analyze`?

Reply:
Yes.
There are test which call analyze and don't provide any assertions to the usecase engine ( e.g. TestOutlineTreeLineNumbers) . So it has null as the testdata, To handle this I placed this method at 2 places,
one at analyze and once here. Calling it here updates the hashmap with the testData